Xamarin.Forms.Platform.iOS/Renderers/PageContainer.cs

```diff
@@ -22,22 +24,40 @@ public PageContainer()
 			IsAccessibilityElement = false;
 		}
 
-		List<NSObject> AccessibilityElements
+		public override bool IsAccessibilityElement
 		{
+			get => false;
+			set => base.IsAccessibilityElement = value;
+		}
+
+		[Internals.Preserve(Conditional = true)]
+		public virtual NSArray AccessibilityElements
+		{
+			[Export("accessibilityElements", ArgumentSemantic.Copy)]
 			get
 			{
+				if (_loaded)
+					return _accessibilityElements;
+				
 				// lazy-loading this list so that the expensive call to GetAccessibilityElements only happens when VoiceOver is on.
 				if (_accessibilityElements == null || _accessibilityElements.Count == 0)
 				{
-					_accessibilityElements = _parent.GetAccessibilityElements();
+					var elements =_parent.GetAccessibilityElements();
+					if(elements != null)
```

Comment:
The documentation for this property says the default is `null`
https://developer.apple.com/documentation/objectivec/nsobject/1615147-accessibilityelements

> Containers can implement this property instead of the dynamic methods to support the retrieval of the contained elements. The default value of this property is nil.

When I do that it seems like the accessibility just returns to the default behavior. I added all the repro's from the PR listed here https://github.com/xamarin/Xamarin.Forms/pull/9702 and I tested the Accessibility demo on xamarin-samples.

We might need to add the setAccessibilityElements side of this as well for certain edge cases. 

For now we'll keep it simple and see if that works